.github/workflows/release.yml

```diff
@@ -0,0 +1,105 @@
+name: Release
+
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
+  repository_dispatch:
+    types:
+    - publish_latest_tag
+    - publish_master
```

Comment:
are you using the dispatch?

Reply:
Not sure what you mean? All I know is that works on my repo (see release link in main body)